pyhf/tensor/mxnet_backend.py

```diff
@@ -324,6 +330,12 @@ def simple_broadcast(self, *args):
                      for arg in args]
         return nd.stack(*broadcast)
 
+    def shape(self, tensor):
+        return tensor.shape
+
```

Comment:
I think for the other backends this is a tuple of ints, here it is long's .. maybe not too important but wanted to point out

Reply:
Not super important if one makes a note of it.